clang/lib/StaticAnalyzer/Core/MemRegion.cpp

```diff
@@ -1068,10 +1068,10 @@ const VarRegion *MemRegionManager::getVarRegion(const VarDecl *D,
     llvm::PointerUnion<const StackFrameContext *, const VarRegion *> V =
       getStackOrCaptureRegionForDeclContext(LC, DC, D);
 
-    if (V.is<const VarRegion*>())
-      return V.get<const VarRegion*>();
+    if (isa<const VarRegion *>(V))
```

Comment:
I wonder if the more canonical way would be to use dyn_cast here and return the result.

Reply:
Switched to `dyn_cast_if_present` because `V` could be null.